source/Set.h

```diff
@@ -91,4 +93,14 @@ void Set<Type>::Revert(const Set<Type> &other)
 
 
 
+template <class Type>
+void Set<Type>::Erase(const std::string &key)
+{
+	const auto it = data.find(key);
+	if(it != data.end())
+		data.erase(it);
```

Comment:
You can replace the whole function with `data.erase(key);`

Reply:
removed the erase function as I dont need it anymore